lib/date-controller.js

```diff
@@ -8,8 +8,9 @@ var _ = require('lodash');
 var dateFormat = 'DD-MM-YYYY';
 var prettyDate = 'D MMMM YYYY';
 
-var DateController = function DateController() {
+var DateController = function DateController(options) {
   Controller.apply(this, arguments);
+  this.options = options || {};
```

Comment:
I know I suggested this but I was thinking that perhaps we should:
```
if (_.isUndefined(this.options)) {
  this.options = options || {};
}
```

It's a little more defensive and means if the BaseClass does something to this.options we don't just overwrite it.